PROJECT_TIERS.md

```diff
@@ -0,0 +1,63 @@
+# Project Tiers
+
+The MapLibre Organization maintains the map rendering frameworks MapLibre GL JS and MapLibre GL Native and fosters an ecosystem of plugins and tools around these libraries to produce the best maps in the world.
+
+The three project tiers Core, Supported, and Hosted describe qualitatively how donated resources are distributed among the projects.
+
+## Tiers
+
+### Core Project
+
+A project that the MapLibre Organization is committed to in a long-term way. A substantial amount of funding is spent on the project in the form of paid Maintainer time and paid Bounties to ensure its long-term success.
+
+### Supported Project
+
+A project that the MapLibre Organization can allocate funding to in the form of paid Maintainer time and paid Bounties. Supported Projects receive a substantially smaller amount of funding than Core Projects and the MapLibre Organization cannot guarantee any level of long-term support.
+
+### Hosted Project
+
+A project that the MapLibre Organization does not allocate any funding to. The project is hosted in the MapLibre GitHub organization to facilitate collaboration between the people who are interested in the project.
+
+## Process
+
+New projects start in the Hosted tier.
+
+If you would like to propose to change the tier of a project, please edit this file in a pull request.
+
+Changing the tier of a project requires approval by the MapLibre Governing Board, as the Board is responsible for all financial decisions.
+
+## List of Projects
+
+### Core
+
+* [maplibre-gl-js](https://github.com/maplibre/maplibre-gl-js)
+* [maplibre-gl-native](https://github.com/maplibre/maplibre-gl-native)
+
+### Supported
+
+* Currently none.
+
+### Hosted
+
+* [maplibre-gl-native-boost](https://github.com/maplibre/maplibre-gl-native-boost)
+* [mbtileserver-rs](https://github.com/maplibre/mbtileserver-rs)
+* [maplibre-plugins-android](https://github.com/maplibre/maplibre-plugins-android)
+* [demotiles](https://github.com/maplibre/demotiles)
+* [maplibre-gl-native-distribution](https://github.com/maplibre/maplibre-gl-native-distribution)
+* [maplibre-gl-directions](https://github.com/maplibre/maplibre-gl-directions)
+* [martin-landing-page](https://github.com/maplibre/martin-landing-page)
+* [maplibre-native-base](https://github.com/maplibre/maplibre-native-base)
+* [maplibre.github.io](https://github.com/maplibre/maplibre.github.io)
+* [awesome-maplibre](https://github.com/maplibre/awesome-maplibre)
+* [qt-geoservices-maplibre-gl](https://github.com/maplibre/qt-geoservices-maplibre-gl)
+* [font-maker](https://github.com/maplibre/font-maker)
+* [maplibre-react-native](https://github.com/maplibre/maplibre-react-native)
+* [maplibre-gl-leaflet](https://github.com/maplibre/maplibre-gl-leaflet)
+* [maplibre-navigation-android](https://github.com/maplibre/maplibre-navigation-android)
+* [maplibre-gl-geocoder](https://github.com/maplibre/maplibre-gl-geocoder)
+* [maplibre-gl-js-docs](https://github.com/maplibre/maplibre-gl-js-docs)
```

Comment:
Our maplibre-gl-js docs should also be eligible to receive maintainer time.

Reply:
I agree with what @birkskyum said.
It might be that the js docs will be part of maplibre-gl-js in the future, but even if not, it requires the maintainer's time.

Reply:
I agree as well. Added maplibre-gl-js-docs to the core projects.

Reply:
I agree as well. Added maplibre-gl-js-docs to the core projects.